test/stacktraces/test/runner.test.ts

```diff
@@ -0,0 +1,41 @@
+import { resolve } from 'pathe'
+import fg from 'fast-glob'
+import { execa } from 'execa'
+import { describe, expect, it } from 'vitest'
+
+describe('stacktraces should respect sourcemaps', async () => {
+  const root = resolve(__dirname, '../fixtures')
+  const files = await fg('*.test.*', { cwd: root })
+
+  for (const file of files) {
+    it(file, async () => {
+      // in Windows child_process is very unstable, we skip testing it
+      if (process.platform === 'win32' && process.env.CI)
+        return
+
+      let error: any
+      await execa('npx', ['vitest', 'run', file], {
+        cwd: root,
+        env: {
+          ...process.env,
+          CI: 'true',
+          NO_COLOR: 'true',
+        },
+      })
+        .catch((e) => {
+          error = e
+        })
+
+      expect(error).toBeTruthy()
+      const msg = String(error)
+        .split(/\n/g)
+        .reduce((acc, line) => {
+          if (line.includes('Start at') || line.includes('Duration') || line.includes('(1 test | 1 failed)') || line.includes(root))
```

Comment:
Maybe we should only include this part in snapshot?

<img width="435" alt="Screenshot 2022-09-13 at 10 36 57" src="https://user-images.githubusercontent.com/16173870/189839841-37faaacb-2a3d-4ac8-9abe-8775d2cdd15d.png">

Reply:
Agree, will do

Reply:
Done, they look nicer now